src/platform/site-wide/sass/schedule-an-appointment.scss

```diff
@@ -0,0 +1,7 @@
+button#crisis-expander-link {
```

Comment:
We have a `va-button-link` class that should turn a button into a correctly styled link. Will that work for your use case?

Reply:
Okay I will try using it.